browser-extension/api-server/Dockerfile

```diff
@@ -6,6 +6,9 @@ RUN cd /app
 RUN npm install
 RUN npm install sequelize-cli
 RUN npm install -g nodemon
+COPY ./entrypoint.sh /
+RUN chmod +x /entrypoint.sh
+ENTRYPOINT ["/entrypoint.sh"]
 COPY . .
 
-CMD ["node", "index.js"]
\ No newline at end of file
+CMD ["node", "index.js"]
```

Comment:
@duggalsu I am not sure what having both keywords ENTRYPOINT and CMD achieve. My understanding is that the latter command will override the former one.

Reply:
CMD is only ignored if the entrypoint is defined in the docker compose file, not the dockerfile. So, both ENTRYPOINT and CMD are ok here.
- https://docs.docker.com/compose/compose-file/compose-file-v3/#entrypoint
- https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#entrypoint